spec/lib/twingly/http_spec.rb

```diff
@@ -552,6 +614,8 @@ class CustomError < StandardError; end
 
   describe "#post", vcr: Fixture.post_example_org do
     include_examples "common HTTP behaviour for", :post, "example.org"
+    include_examples "verifies proxy functionality", :post
```

Comment:
This isn't needed anymore if you make the changes in my comment above.

```suggestion
    include_examples "verifies proxy functionality"
```